airflow/pyspark/dags/config/submit_dataproc_template_serverless.ini

```diff
@@ -0,0 +1,33 @@
+[COMPOSER]
+DAG_NAME=submit_dataproc_template_serverless
+SCHEDULE_INTERVAL=@hourly
```

Comment:
Should we add capability to run it one-time as well, without any scheduling like `SCHEDULEING=true / false`

Reply:
Added documentation on how to run without schedule